applications/metadata.js

```diff
@@ -175,7 +175,7 @@ var metadata = { "applications":
             "isActive": true,
             "directory": "DANCE",
             "name": "Dance App",
-            "description": "Dance Like nobody is watching.<br> You can que up different Dance animations and let Your Avatar do all the dancing while yoyu sit back and relax.",
+            "description": "Dance Like nobody is watching.<br> You can Queue up different Dance animations and let Your Avatar do all the dancing while yoyu sit back and relax.",
```

Comment:
Missed the other one haha;

Proposed change:

`"description": "Dance Like nobody is watching.<br>You can queue up different dance animations and let your avatar do all the dancing while you sit back and relax.",`